gulpfile.js

```diff
@@ -257,4 +240,4 @@ gulp.task('watch', ['build'], function() {
 gulp.task('build', ['sass', 'webfont', 'copy:fonts', 'processHtml', 'copy:images']);
 
 // Define Dist generation task (Deploy)
-gulp.task('dist', ['sass:dist', 'fonts:dist', 'processHtml:dist', 'images:dist']);
+gulp.task('dist', ['sass:build', 'fonts:dist', 'processHtml:dist', 'images:dist']);
```

Comment:
You are using sass:build tasks that has sourcemaps build, I think it's best to leave sass:dist here.

Reply:
You're right, the change I made does not correspond. I changed it because I was not copying the css  the dev folder to the dist folder. Thanks for the gesture.